packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -438,24 +402,20 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
       created_at: {
         type: "TIMESTAMP WITH TIME ZONE",
         notNull: true,
+        default: pgm.func("clock_timestamp()"),
       },
     },
     {
       ifNotExists: true,
     },
   );
-  // Currently links are between unversioned entities.
-  // Ideally we'd have links between versioned entities -> unversioned entities.
-  pgm.addConstraint("links", "links_pkey", {
-    primaryKey: [
-      "source_entity_id",
-      "target_entity_id",
-      "link_type_version_id",
-    ],
+  // Currently entity relations are between unversioned entities.
+  pgm.addConstraint("entity_relations", "entity_relations_pkey", {
+    primaryKey: ["source_entity_id", "link_entity_id", "target_entity_id"],
```

Comment:
I guess these actually _are_ good enough as primary keys now? As this is immutable and doesn't have any sense of versioning, just the actual link entity has a concept of archived or not.

I do wonder if we want to call this source and target, or if we want left and right or something. Just in case we come to inverse or bidirectional links... (Or would we store two entity relations for those)

Reply:
I definitely think the primary key here is better than before. Do we want to introduce a pseudo ID to these, or are we okay with addressing the relation by all the entity ids that are part of the link?

Reply:
After our discussion this morning I don't think this matters :)